tests/Validation/ValidationDateRuleRest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Illuminate\Tests\Validation;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Validation\Rules\Date;
+
+class ValidationDateRuleRest extends TestCase
+{
+    public function testDate()
+    {
+        $rule = new Date();
+        $this->assertEquals('date', (string) $rule);
```

Comment:
Changing to `assertSame` isn't providing any benefit in this context. We are specifically casting the object to a string and comparing against a string. Therefore, we already know their types are equivalent.